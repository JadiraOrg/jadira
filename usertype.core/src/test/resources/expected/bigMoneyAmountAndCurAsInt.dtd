<!ELEMENT dataset (
    BIGMONEYAMOUNTANDCURASINT*)>

<!ELEMENT BIGMONEYAMOUNTANDCURASINT EMPTY>
<!ATTLIST BIGMONEYAMOUNTANDCURASINT
    ID CDATA #REQUIRED
    MY_CURRENCY CDATA #IMPLIED
    MY_AMOUNT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

