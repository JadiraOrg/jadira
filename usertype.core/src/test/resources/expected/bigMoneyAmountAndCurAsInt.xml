<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "bigMoneyAmountAndCurAsInt.dtd">
<dataset>
  <BIGMONEYAMOUNTANDCURASINT ID="0" MY_CURRENCY="840" MY_AMOUNT="100.00" NAME="test_0"/>
  <BIGMONEYAMOUNTANDCURASINT ID="1" MY_CURRENCY="840" MY_AMOUNT="100.10" NAME="test_1"/>
  <BIGMONEYAMOUNTANDCURASINT ID="2" MY_CURRENCY="978" MY_AMOUNT="0.99" NAME="test_2"/>
  <BIGMONEYAMOUNTANDCURASINT ID="3" MY_CURRENCY="978" MY_AMOUNT="-0.99" NAME="test_3"/>
  <BIGMONEYAMOUNTANDCURASINT ID="4" NAME="test_4"/>
</dataset>